packit/specfile.py

```diff
@@ -80,12 +80,6 @@ def set_spec_version(
             if not changelog_entry:
                 return
 
-            if not self.spec_content.section("%changelog"):
```

Comment:
do we have a test case which covers this?

Reply:
iirc it was added with a test case… can improve it if needed, will have a look

Reply:
thanks! I recall I wrote a test case for that, glad the code is no longer needed